tests/crates/enarx_exec_tests/src/bin/sgx_attestation.rs

```diff
@@ -39,11 +39,22 @@ impl TryFrom<u64> for TeeTech {
     }
 }
 
+#[derive(Sequence)]
+pub struct CrlPair<'a> {
+    pub url: String,
+    pub crl: CertificateList<'a>,
+}
+
+#[derive(Sequence)]
+pub struct CachedCrl<'a> {
+    pub crls: Vec<CrlPair<'a>>,
+}
+
```

Comment:
You should be importing `CrlList` structure from `enarx`

Reply:
I'd have to add `enarx = { path = "../../../" }` to `tests/crates/enarx_exec_tests/Cargo.toml`, is that right? SEV structures were copied over for `sev_attestation.rs`, so there's a precedent for copying the structure over.

Reply:
Also, Enarx `src/` only has `main.rs`, no `lib.rs`.

Reply:
Copied to the `lib.rs` in the exec tests crate, created an issue to track moving some common items to a common items crate in Enarx, https://github.com/enarx/enarx/issues/2451.